butterknife-lint/src/main/java/butterknife/lint/LintRegistry.java

```diff
@@ -13,4 +13,8 @@
   @Override public List<Issue> getIssues() {
     return ImmutableList.of(InvalidR2UsageDetector.ISSUE);
   }
+
+  @Override public int getApi() {
+    return com.android.tools.lint.detector.api.ApiKt.CURRENT_API;
```

Comment:
Import?

Reply:
haha, i copied it straight from the lint warning. fixed!